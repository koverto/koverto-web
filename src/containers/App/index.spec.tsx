import * as React from "react"
import { render } from "@testing-library/react"
import { App } from "."

describe("App", () => {
  it("matches the snapshot", () => {
    const { container } = render(<App />)
    expect(container).toMatchSnapshot()
  })
})
