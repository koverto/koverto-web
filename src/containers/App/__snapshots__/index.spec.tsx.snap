// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches the snapshot 1`] = `
<div>
  <h1>
    Dashboard
  </h1>
</div>
`;
